pkg/cli/cli.go

```diff
@@ -569,18 +569,22 @@ func runCreateCVDCommand(c *cobra.Command, flags *CreateCVDFlags, opts *subComma
 		flags.CreateCVDOpts.Host = ins.Name
 	}
 	statePrinter.Print(createCVDStateMsg)
-	cvd, err := createCVD(service, *flags.CreateCVDOpts)
+	cvds, err := createCVD(service, *flags.CreateCVDOpts)
 	statePrinter.PrintDone(createCVDStateMsg)
 	if err != nil {
 		return err
 	}
-	statePrinter.Print(connectCVDStateMsg)
-	cvd.ConnStatus, err = ConnectDevice(flags.CreateCVDOpts.Host, cvd.Name, &command{c, &flags.Verbose}, opts)
-	statePrinter.PrintDone(connectCVDStateMsg)
-	if err != nil {
-		err = fmt.Errorf("Failed to connect to device: %w", err)
+	for _, cvd := range cvds {
+		statePrinter.Print(fmt.Sprintf(connectCVDStateMsgFmt, cvd.Name))
+		cvd.ConnStatus, err = ConnectDevice(flags.CreateCVDOpts.Host, cvd.Name, &command{c, &flags.Verbose}, opts)
+		statePrinter.PrintDone(fmt.Sprintf(connectCVDStateMsgFmt, cvd.Name))
+		if err != nil {
+			err = fmt.Errorf("Failed to connect to device: %w", err)
+		}
+	}
+	for _, cvd := range cvds {
```

Comment:
Any reason this is done in a separate loop?

Reply:
yes, form a UX perspective I'd like to see the order of events printed first and then the list of devices, e.g

```
Creating Host ... Done
Creating CVD ... Done
Connecting to cvd-1 ... Done
Connecting to cvd-2 ... Done

cvd-1 (cf-c8774d61-9197-4c1d-9f4f-3b4dd4bcd510)
  Status: Running
  ADB: connected
  Displays: [720 x 1280 ( 320 )]
  WebRTCStream: http://foo.com/client.html
  Logs: http://foo.com/logs/

cvd-2 (cf-c8774d61-9197-4c1d-9f4f-3b4dd4bcd510)
  Status: Running
  ADB: connected
  Displays: [720 x 1280 ( 320 )]
  WebRTCStream: http://foo.com/client.html
  Logs: http://foo.com/logs/
```